src/olympia/amo/log.py

```diff
@@ -234,7 +234,9 @@ class REQUEST_SUPER_REVIEW(_LOG):
     short = _(u'Super review requested')
     keep = True
     review_queue = True
-    hide_developer = True
+    sanitize = _(u'The addon has been flagged for Admin Review.  Its still in '
+                 u'our review queue, but it will need to be checked by one of '
+                 u'our admin reviewers.')
```

Comment:
I cribbed it from https://github.com/mozilla/addons-server/blob/master/src/olympia/editors/templates/editors/emails/author_super_review.ltxt but tried to make it a bit shorter.  Alternative copy suggestion?